README.md

```diff
@@ -78,6 +78,23 @@ The `cosi` application provides both a client for signing messages,
 and a server implementing the cosigner or witness-server role
 in the CoSi protocol.
 
+## Chose the right directory for the examples
+
+For the examples in the following sections, we suppose you're in a directory
+where you can find the following files: `README.md` and `dedis-group.toml`.
+
+If you compiled from source, please change the directory like so:
+
+```bash
+cd $GOPATH/src/github.com/dedis/cosi
+```
+
+If you used the binary distribution, please use
+
+```bash
+cd $( dirname $( which cosi ) )
```

Comment:
If ~/bin is not the in $PATH, then it won't work. Or if the user extracted it elsewhere, it won't work too. Maybe we should add a sentence like 
```
If you used the the binary distribution, please use : 
....
Or if you extracted cosi not in ~/bin, go to the extracted folder
```
something along those lines ?